app/assets/javascripts/mortgage_calculator/angular/services/stamp_duty.js

```diff
@@ -69,12 +46,15 @@ App.factory('StampDuty', function() {
               remainingTaxable = 0,
               bandTaxable = 0;
 
+          if(rateObj.threshold === null) {
+              rateObj.threshold = Infinity;
+          }
```

Comment:
I understand the code but not for what is for. What is the purpose of set the threshold to Infinity? Can you add the answer to my question to the commit message body, please?

Reply:
When a band has no upper limit its threshold value could be considered to be infinite. This simplifies calculations that check the upper limit of a threshold.

Do you think this still needs describing in the commit body?

Reply:
It is alway good to fill the commit messages with much information as possible, so I would say yes.